### DISCUSSION TITLE
USB JMS562 does not start

### DISCUSSION THREAD

**Discussion by gek2** [dorssel/usbipd-win#400] [Q&A] [answered]
help to connect UAS driver. hdd 3Tb not detected
![Screenshot 2022-05-30 221148](https://user-images.githubusercontent.com/53976053/171049261-27a06b04-5618-4828-8e23-93bf7341041e.jpg)
![Screenshot 2022-05-30 221406](https://user-images.githubusercontent.com/53976053/171049427-707840fd-3b3a-4857-966a-3f65325b39aa.jpg)
Thanks

**Comment by dorssel** [comment, marked as answer]
The default WSL kernel does not have support for USB storage devices, see `/proc/config.gz`:

```
# CONFIG_USB_STORAGE is not set
```

You will need to compile your own kernel with USB storage support. See https://github.com/dorssel/usbipd-win/wiki/WSL-support#building-your-own-usbip-enabled-wsl-2-kernel.
